### PR TITLE
LPS-146582: Missing Highlight

### PR DESCRIPTION
**Pull request by peerkar**
https://issues.liferay.com/browse/LPS-146582

### CODE REVIEW

**Comment by peerkar** [comment]
ci:test:sf

**Comment by lipusz** [comment]
ci:test:search

File: modules/dxp/apps/search-experiences/search-experiences-service/src/main/java/com/liferay/search/experiences/internal/blueprint/search/request/body/contributor/HighlightSXPSearchRequestBodyContributor.java [hunk 1]
```diff
@@ -40,7 +40,9 @@ public void contribute(
 		HighlightConfiguration highlightConfiguration =
 			configuration.getHighlightConfiguration();
 
-		if (highlightConfiguration == null) {
+		if ((highlightConfiguration == null) ||
```
Comment (lipusz) [inline modules/dxp/apps/search-experiences/search-experiences-service/src/main/java/com/liferay/search/experiences/internal/blueprint/search/request/body/contributor/HighlightSXPSearchRequestBodyContributor.java:43]: Do other contributors suffer from the same issue? Like Aggregations?

Comment (peerkar) [reply modules/dxp/apps/search-experiences/search-experiences-service/src/main/java/com/liferay/search/experiences/internal/blueprint/search/request/body/contributor/HighlightSXPSearchRequestBodyContributor.java:43]: I didn't find others to have the same issue

**Comment by lipusz** [comment]
Looks like there are some related failures.

**Comment by peerkar** [comment]
ci:test:search

**Comment by peerkar** [comment]
ci:test:sf

**Comment by peerkar** [comment]
ci:forward

**Comment by peerkar** [comment]
@bchan this pull has no unique or related failures but are legacy of https://issues.liferay.com/browse/LPS-143238

**Comment by lipusz** [comment]
ci:forward

**Comment by pyoo47** [comment]
ci:stop

Stopping and restarting due to ci-infrastructure bug

**Comment by pyoo47** [comment]
ci:test:relevant

**Comment by peerkar** [comment]
ci:stop

**Comment by peerkar** [comment]
ci:test:search

**Pull request closed without merging**